homework_4/02_bets_api/internal/domain/services/betService.go

```diff
@@ -0,0 +1,33 @@
+package services
+
+import (
+	"context"
+	"github.com/pkg/errors"
+	"github.com/superbet-group/code-cadets-2021/homework_4/02_bets_api/internal/api/controllers/models"
+	"github.com/superbet-group/code-cadets-2021/homework_4/02_bets_api/internal/infrastructure/sqlite"
+)
+
+type BetService struct {
+	betRepository sqlite.BetRepository
```

Comment:
This shouldn't directly depend on `sqlite.BetRepository`, create an interface for it!